index.js

```diff
@@ -48,16 +48,21 @@ module.exports = {
 
   // we have to use the outputReady hook to ensure that ember-cli has finished copying the contents to the outputPath directory
   outputReady(result) {
-    // NOTE: result.directory is not updated and still references the same path as postBuild hook (this might be a bug in ember-cli)
-    // Set the distPath to the "final" outputPath, where EMBER_CLI_TEST_OUTPUT is the path passed to testem (set by ember-cli).
-    // We fall back to result.directory if EMBER_CLI_TEST_OUTPUT is not set.
-    let distPath = process.env.EMBER_CLI_TEST_OUTPUT || result.directory;
-    let { pkg } = this.project;
-
-    if (this.fastboot) {
-      reloadServer(this.fastboot, distPath);
-    } else {
-      this.fastboot = createServer(distPath, pkg);
+    const isEnabled =
+      this.app.name === 'dummy' || this.app.env !== 'production';
```

Comment:
Just a thought that...

While technically correct to check if `this.app.env !== 'production'` a number of apps still incorrectly use custom env names which would cause a false positive here.

🤔 Wonder if enabling/disabling this addon could be user configurable with sensible defaults (much like in `ember-cli-mirage`.

Reply:
@gilest that's interesting idea. However, I think there is noticeable difference with `ember-cli-mirage`:
you may use `ember-cli-mirage` during `ember s` to mock backend server, whereas `ember-cli-fastboot-testing` makes sense *only* for testing purposes and it may not even needed for `app.env == development` use case

Reply:
we could have a separate PR with such enhancement

Reply:
Agree on both points. Believe this would satisfy:

```suggestion
    const isEnabled =
      this.app.name === 'dummy' || this.app.env === 'test';
```